pkg/security/api/v1/types.go

```diff
@@ -0,0 +1,86 @@
+package v1
+
+import (
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	kapi "k8s.io/kubernetes/pkg/api/v1"
+)
+
+// PodSecurityPolicySubjectReview checks whether a particular user/SA tuple can create the PodSpec.
+type PodSecurityPolicySubjectReview struct {
+	unversioned.TypeMeta `json:",inline"`
+
+	// Spec defines specification for the PodSecurityPolicySubjectReview.
+	Spec PodSecurityPolicySubjectReviewSpec `json:"spec"`
+
+	// Status represents the current information/status for the PodSecurityPolicySubjectReview.
+	Status PodSecurityPolicySubjectReviewStatus `json:"status,omitempty"`
+}
+
+// PodSecurityPolicySubjectReviewSpec defines specification for PodSecurityPolicySubjectReview
+type PodSecurityPolicySubjectReviewSpec struct {
+	// PodSpec is the PodSpec to check.
+	PodSpec kapi.PodSpec `json:"podSpec"`
+
+	// User is the user you're testing for.
```

Comment:
Be use to use lower case in the names - since the godoc is used in swagger here, you'll want to refer to the field name as it would be in json.  I.e. `user is the user you're testing for` and `if you specify "user" but not "group"`.